src/graph_gui.cpp

```diff
@@ -165,8 +165,11 @@ struct ValuesInterval {
 struct BaseGraphWindow : Window {
 protected:
 	static const int GRAPH_MAX_DATASETS     =  64;
-	static const int GRAPH_AXIS_LINE_COLOUR =  GREY_SCALE(1);
+	static const int GRAPH_BASE_COLOUR      =  GREY_SCALE(2);
+	static const int GRAPH_GRID_COLOUR      =  GREY_SCALE(3);
+	static const int GRAPH_AXIS_LINE_COLOUR =  GREY_SCALE(1); ///< Black
```

Comment:
Weird comment, "Black". What does it add / what was the intention?

Reply:
Black is 1, not 0. This explanation seemed necessary at midnight, not so much now. :)